test/integration/integration_test.go

```diff
@@ -89,7 +89,7 @@ func testBasicBaseConfig(grpcPort, perSecond string, local_cache_size string) fu
 		os.Setenv("RUNTIME_SUBDIRECTORY", "ratelimit")
 		os.Setenv("REDIS_PERSECOND_SOCKET_TYPE", "tcp")
 		os.Setenv("REDIS_SOCKET_TYPE", "tcp")
-		os.Setenv("LOCAL_CACHE_SIZE", local_cache_size)
+		os.Setenv("LOCAL_CACHE_SIZE_IN_BYTES", local_cache_size)
```

Comment:
In #111 (comment), we updated the env variable name from LOCAL_CACHE_SIZE to LOCAL_CACHE_SIZE_IN_BYTES, but missed updating the env variable name in test case. (It is a duplicate change for https://github.com/lyft/ratelimit/pull/112)

Reply:
Why don't the integration tests run in CI? I don't remember the history here. Can you fix this? cc @junr03

Reply:
The integration tests run in the CI. In current framework of integration test, client sends the requests, and then verifies the response.  When adding the integration test for local_cache, we expect the responses are same as before, no matter whether local_cache is enabled or not. if there is a typo for env variable, it means the local_cache is disabled. That's why it is not easy to catch the typo. But there is one trick we can do to verify whether local_cache is used in the integration test: add time.sleep into it,, check {debug_port}/stats to see local_cache related stats.

Reply:
it all runs in the same travis target: https://travis-ci.org/lyft/ratelimit/builds/631398436#L480. It could be separated into a unit test and an integration test target.

Reply:
@junr03 , but if I understood correctly, separating it into another target won't help to catch this typo. By mistyping `LOCAL_CACHE_SIZE_IN_BYTES ` as `LOCAL_CACHE_SIZE `, it means the local_cache is disabled. Since we expect the responses are same no matter whether local_cache is enabled or not, the tests on new-added target still pass.

Reply:
Yep, I wasn't talking about the typo problem. I was talking about making it easier to see that there are CI targets for both unit tests and integration tests.

Reply:
I'm hesitating to make that change. Generally, the developer should be able to setup the unit test and integration test env, and figure out which part goes wrong if the test fails.